modules/server/src/main/scala/navigate/server/NavigateEngine.scala

```diff
@@ -370,10 +426,11 @@ object NavigateEngine {
     enableGuide = false,
     disableGuide = false,
     oiwfsObserve = false,
-    oiwfsStopObserve = false
+    oiwfsStopObserve = false,
+    guideConfig = GuideConfig.defaultGuideConfig
   )
 
-  private def command[F[_]: MonadThrow: Logger](
+  private def simpleCommand[F[_]: MonadThrow: Logger](
```

Comment:
This is used for simple commands, just an `F` that executes the command when evaluated, producing a result. The method takes care of setting/releasing guard flags in the global state and surrounding the evaluation with log messages.
An important distinction with the new `command` method is that here, the command code does not have access to the global state.

Reply:
👍